contracts/deploy/10_token.ts

```diff
@@ -7,6 +7,7 @@ const func: DeployFunction = async function (hre: HardhatRuntimeEnvironment) {
   const { deployer } = await getNamedAccounts();
 
   await deploy("SimpleToken", { from: deployer, log: true });
+  await deploy("MaliciousToken", { from: deployer, log: true });
```

Comment:
Is it useful to deploy the malicious token with hardhat? If yes, would it be better to put it into it's own deployment file so we can still do a deployed with `SimpleToken` but without `MaliciousToken` if needed?

Reply:
Yes, a new script for the Malicious Token has been added in 51cd9d0bc1b1accc21ce6f172e09fae451bee9c4.